content/chainguard/administration/custom-idps/custom-idps/index.md

```diff
@@ -143,7 +139,7 @@ Next, use `chainctl` to log in to Chainguard with an OIDC provider (such as Goog
 chainctl auth login
 ```
 
-This bootstrap account can be used as a [backup account](/chainguard/administration/custom-idps/custom-idps/#backup-accounts) (that is, a backup account you can use to log in if you ever lose access to your primary account). However, if you prefer to remove this role-binding after configuring the custom IDP, you may also do so.
+This bootstrap account — which can be any personal Chainguard account with access to your organization — can be used as a [backup account](/chainguard/administration/custom-idps/custom-idps/#backup-accounts) (that is, a backup account you can use to log in if you ever lose access to your primary account). However, you can also disable this account by removing the role-binding.
```

Comment:
did you mean to write this? I'm a little confused now :)

I thought you would write something like the following, but I may have misunderstood

> The bootstrap account can use any supported IDP -- for example you may choose to temporarily use a personal Google account. You can leave this account active as a backup account (link) or, if you prefer, you can delete the account by removing the role-binding after configuring the custom IDP.

Reply:
I think I was a little confused myself!